.github/workflows/build-book.yaml

```diff
@@ -80,9 +86,10 @@ jobs:
         uses: dawidd6/action-download-artifact@v3.1.4
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          workflow: ${{ inputs.trigger_workflow }}
+          workflow: ${{ inputs.workflow }}
           run_id: ${{ github.event.workflow_run.id }}
           name: ${{ inputs.code_artifact_name }}
+          workflow_conclusion: ${{ inputs.workflow_conclusion }}
```

Comment:
Right, so I understand why this is here and don't really have a better suggestion, but I want to note that there could be bugs if either the default `workflow_conclusion` input is used or if a a user specifies all of the available inputs here.

Reply:
Thanks Anissa. I'll keep this in mind if things pop up.